step.sh

```diff
@@ -4,7 +4,7 @@ THIS_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 tmp_gopath_dir="$(mktemp -d)"
 
-go_package_name="github.com/bitrise-io/steps-xcode-archive"
```

Comment:
pls remove this file, since we use the Go toolkit to perform the Go script

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/commits/78a85e677b3e813fe5420c2aaa334e1030510ebd